README.markdown

```diff
@@ -1,6 +1,6 @@
 Build Status
 ------------
-[![Build Status](https://travis-ci.org/php-memcached-dev/php-memcached.png)](https://travis-ci.org/php-memcached-dev/php-memcached)
+![Build Status](https://github.com/php-memcached-dev/php-memcached/actions/workflows/build-and-test.yml/badge.svg?branch=master)
```

Comment:
Maybe merged too soon, I'd suggest linking to `https://github.com/php-memcached-dev/php-memcached/actions` as the target of this image

Reply:
I found on StackOverflow an example `[<img src="path/to/image.png">](https://link-to-your-URL/)` -- I don't know if it will work for an absolute img src url vs. relative path within the repo. Apparently the syntax for this changed in 2023!